src/contracts/misc/interfaces/IGhoStewardV2.sol

```diff
@@ -0,0 +1,147 @@
+   * - the facilitator is controlled
+   * @dev Only callable by Risk Council
+   * @param facilitator The facilitator address
+   * @param newBucketCapacity The new facilitator bucket capacity
+   */
+  function updateFacilitatorBucketCapacity(address facilitator, uint128 newBucketCapacity) external;
+
+  /**
+   * @notice Updates the borrow rate of GHO, only if:
+   * - respects the debounce duration (7 day pause between updates must be respected)
+   * - the update changes up to 0.5% upwards
+   * - the update is lower than 9.5%
+   * @dev Only callable by Risk Council
+   * @param newBorrowRate The new variable borrow rate (expressed in ray) (e.g. 0.0150e27 results in 1.50%)
+   */
+  function updateGhoBorrowRate(uint256 newBorrowRate) external;
+
+  /**
+   * @notice Updates the borrow rate of GHO, only if:
+   * - respects the debounce duration (7 day pause between updates must be respected)
+   * - the GSM address is approved
+   * - the update changes up to 100% upwards
+   * @dev Only callable by Risk Council
+   * @param gsm The gsm address to update
+   * @param newExposureCap The new exposure cap
+   */
+  function updateGsmExposureCap(address gsm, uint128 newExposureCap) external;
+
+  /**
+   * @notice Updates the borrow rate of GHO, only if:
+   * - respects the debounce duration (7 day pause between updates must be respected)
+   * - the GSM address is approved
+   * - the update changes up to 50 bps (0.5%) upwards (for both buy and sell individually);
+   * @dev Only callable by Risk Council
+   * @param gsm The gsm address to update
+   * @param buyFee The new buy fee (expressed in bps) (e.g. 0.0150e4 results in 1.50%)
+   * @param sellFee The new sell fee (expressed in bps) (e.g. 0.0150e4 results in 1.50%)
+   */
+  function updateGsmFeeStrategy(address gsm, uint256 buyFee, uint256 sellFee) external;
+
+  /**
+   * @notice Adds/Removes controlled facilitators
+   * @dev Only callable by owner
+   * @param facilitatorList A list of facilitators addresses to add to control
+   * @param approve A boolean to control or remove control towards the facilitators
+   */
+  function controlFacilitators(address[] memory facilitatorList, bool approve) external;
+
+  /**
+   * @notice Returns the list of controlled facilitators.
+   * @return An array of GSM addresses
+   */
+  function getControlledFacilitators() external view returns (address[] memory);
+
+  /**
+   * @notice Returns the GHO timelock value for borrow rate updates
+   * @return The time of the last GHO borrow rate update (in seconds).
+   */
+  function getGhoBorrowRateTimelock() external view returns (uint40);
+
+  /**
+   * @notice Returns the Gsm timelocks values for all parameters updates
+   * @param gsm The GSM address
+   * @return The GsmDebounce struct with parameters' timelock
+   */
+
+  function getGsmTimelocks(address gsm) external view returns (GsmDebounce memory);
+
+  /**
+   * @notice Returns the facilitator timelock value for bucket capacity updates
```

Comment:
a bit confusing. Would rephrase this, saying it returns the timestamp of the last update